nixos/modules/services/networking/syncthing.nix

```diff
@@ -7,6 +7,21 @@ let
   cfg = config.services.syncthing;
   defaultUser = "syncthing";
```

Comment:
Ah, so there *is* an entry for syncthing in `nixos/modules/misc/ids.nix`. Very cool, thanks for clarifying.